### PR TITLE
Added events, comments and TODOs

### PR DESCRIPTION
**Pull request by GriffGreen**

### CODE REVIEW

File: contracts/LPVault.sol [hunk 1]
```diff
@@ -42,45 +62,50 @@ contract LPVault is Owned {
...
-    ///  instance to this LPvault. Keep in mind this isn't a single pledge but
-    ///  instead an entire liquid pledging contract.
+    /// @notice `onlyOwner` used to attach a specific liquidPledging instance
+    ///  to this LPvault; keep in mind that once a liquidPledging contract is 
+    ///  attached it cannot be undone, this vault will be forever connected
     /// @param _newLiquidPledging A full liquid pledging contract
     function setLiquidPledging(address _newLiquidPledging) public onlyOwner {
         require(address(liquidPledging) == 0x0);
         liquidPledging = LiquidPledging(_newLiquidPledging);
+        VaultSet(_newLiquidPledging);
```
Comment (perissology) [inline contracts/LPVault.sol]: why is this necessary? This will most always be the second call to the vault, so it doesn't seem like we need to log this

File: contracts/LPVault.sol [hunk 2]
```diff
@@ -42,45 +62,50 @@ contract LPVault is Owned {
...
-    /// @param _ref This parameter is used to reference details about the
-    ///  payment from another contract.
-    /// @param _dest This is the address that payments will end up being sent to
-    /// @param _amount This is the amount that the payment is being authorized
-    ///  for.
+    /// @notice `onlyLiquidPledging` authorizes payments from this contract, if 
+    ///  `autoPay == true` the transfer happens automatically `else` the `owner`
+    ///  must call `confirmPayment()` for a transfer to occur (training wheels);
+    ///  either way, a new payment is added to `payments[]` 
+    /// @param _ref References the payment can be a URL, IPFS hash, etc
```
Comment (perissology) [inline contracts/LPVault.sol]: hmm, well what ever is used here will be an argument to `liquidPledging.cancelPayment` and `liquidPledging.confirmPayment` call in the `doConfirmPayment` and `doCancelPayment` vault methods. Since this is the `LPVault` and meant to work with `liquidPledging`, this will be the `pledgeId`

File: contracts/LPVault.sol [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
+
+/// @dev This contract holds ether securely for liquid pledging systems; for
+///  this iteration the funds will come often be escaped to the Giveth Multisig
+///  (safety precaution), but once fully tested and optimized this contract will
 ///  be a safe place to store funds equipped with optional variable time delays
-///  to allow for an optional escape hatch to be implemented
-import "./Owned.sol";
+///  to allow for an optional escapeHatch to be implemented in case of issues;
+///  future versions of this contract will be enabled for tokens
+import "./Owned.sol";// TODO IMPORT ESCAPABLE :-D...............................???????????????????????????/
```
Comment (perissology) [inline contracts/LPVault.sol]: work is done for this. PR coming

File: contracts/LPVault.sol [hunk 4]
```diff
@@ -19,21 +39,21 @@ contract LiquidPledging {
 
 /// @dev `LPVault` is a higher level contract built off of the `Owned`
 ///  contract that holds funds for the liquid pledging system.
-contract LPVault is Owned {
+contract LPVault is Owned {// TODO NEEDS TO BE ESCAPABLE!!! AND WE NEED TO ADD A `WITHDRAW PARTIAL` FUNCTION????????????????
```
Comment (perissology) [inline contracts/LPVault.sol]: done. PR coming

File: contracts/LPVault.sol [hunk 5]
```diff
@@ -19,21 +39,21 @@ contract LiquidPledging {
...
-    bool public autoPay; // if false, payments will take 2 txs to be completed
+    LiquidPledging public liquidPledging; // LiquidPledging contract's address
+    bool public autoPay; // If false, payments will take 2 txs to be completed
 
     enum PaymentStatus {
-        Pending, // means the payment is awaiting confirmation
-        Paid,    // means the payment has been sent
-        Canceled // means the payment will never be sent
+        Pending, // When the payment is awaiting confirmation
+        Paid,    // When the payment has been sent //TODO CHANGE TO SENT?????????????????????????????????
```
Comment (perissology) [inline contracts/LPVault.sol]: either way. I think it's just semantics. They both seem clear to me

Comment (GriffGreen) [reply contracts/LPVault.sol]: Its mostly to avoid having 2 similar enum's with `Paid` we could change the other one instead? or if we change the name of the other enum then we are probably fine.

File: contracts/LPVault.sol (hunk 1 above)
Comment (GriffGreen) [reply contracts/LPVault.sol]: It was a thought, I'm down to kill it. My thinking was that it is a pretty important piece and its something that we would want transparently declared, but since it can only be done once, I'll take it out.
